### DISCUSSION TITLE
Windows/macOS Support

### DISCUSSION THREAD

**Discussion by aaronbr00ks** [invertase/react-native-firebase#4319] [Ideas] [answered]
React Native Firebase should support [React Native Windows](https://github.com/microsoft/react-native-windows) and [React Native macOS](https://github.com/microsoft/react-native-macos) so that apps built with React Native Firebase can work across all react native platforms.

Are there any plans to support these platforms?

**Comment by andersonaddo** [comment]
Probably not; especially since it's low priority and would really a lot of work. 
PRs are welcome, however :)

**Comment by mikehardy** [comment, marked as answer]
Windows unlikely. I believe catalyst works, we have a record of merging things there: https://github.com/invertase/react-native-firebase/issues?q=is%3Aissue+catalyst+is%3Aclosed

This will *definitely* be community driven though - I don't believe anyone else is targeting these platforms at the moment and it is troublesome to support them since react-native core is on record as not wanting platforms other than android and ios in core. For cross-platform development beyond android and ios I must regretfully refer you to Flutter, where you may find the FlutterFire package of interest

**Reply by Salakar** [reply]
For context we also run the show over at FlutterFire so you can expect the same level of quality.

If you're working on a new project and being on as many platforms as possible is critical to your project then Flutter is the better choice, you get Web, Android, iOS, MacOS, Linux & Windows out the box - first-party supported. FlutterFire supports Android, iOS, MacOS, Web on quite a few of the main packages, and we're actively expanding the platform support.

React Native is still great but expanding beyond Android & iOS then you have to rely on platform support implemented by third-parties.
